ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-stable-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -62,6 +62,13 @@ releases:
       version_bounds:
         lower: 4.12.0-0
         upper: 4.13.0-0
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks